hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -423,8 +417,9 @@ private long calculateMaxSteps(BalancerClusterState cluster) {
             maxSteps);
       }
     }
-    LOG.info("start StochasticLoadBalancer.balancer, initCost=" + currentCost + ", functionCost="
-        + functionCost() + " computedMaxSteps: " + computedMaxSteps);
+    LOG.info("Start StochasticLoadBalancer.balancer, initial weighted average imbalance ="
+      + currentCost / sumMultiplier + ", functionCost=" + functionCost() + " computedMaxSteps: "
+      + computedMaxSteps);
```

Comment:
why is `currentCost` divided by `sumMultiplier` here? isn't `currentCost` by itself already the weighted average measure of imbalance?

Reply:
No, it is the weighted sum.